PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,24 @@
+<!-- Check all that apply [x] -->
+
+# Checklist
+
+This is a checklist that can be used in effort to review before submitting a PR, in the Edge Drivers Repo.
+
+- [ ] This is the first time I am contributing code:
+     - [ ] I have reviewed the README.md file
+     - [ ] I have reviewed the CODE_OF_CONDUCT.md file
+     - [ ] I have signed the CLA
+     - [ ] This is a request to add a device for WWST certification and I plan on entering a console request or have entered a console request at developer.smartthings.com
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code in hard-to-understand areas
+- [ ] I have verified my changes with testing with a device or have a plan for testing that has been communicated appropriately 
+- [ ] I am adding new behavior, such as adding a sub-driver, and have added new unit tests to cover the new behavior
+
+
+# Type of change
```

Comment:
Maybe we could add a step here to add the appropriate label to the PR based on what kind of change it is (we already have a WWST label, for example)

Reply:
I am seeing if we can automate this too.